auto_inertial_params/proposal.md

```diff
@@ -0,0 +1,149 @@
+<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML' async></script>
+
+# Automatically Compute Inertia Tensor Proposal
+
+* **Authors**:
+Jasmeet Singh `<jasmeet0915@gmail.com>`, Addisu Taddese `<addisu@openrobotics.org>`, Dharini Dutia `<dharinidutia@openrobotics.org>`
+* **Status**: Draft
+* **SDFormat Version**: 1.11
+* **`libSDFormat` Version**: 14.X 
+
+## Introduction
+
+This proposal suggests adding new attributes and elements to support the automatic calculation of [Moments of Inertia and Products of Inertia](https://en.wikipedia.org/wiki/List_of_moments_of_inertia) for a link in SDFormat 1.11. It also proposes adding support for parsing these elements and attributes in libsdformat14.
+
+Setting physically plausible values for inertial parameters is crucial for an accurate simulation. However, these parameters are often complex to comprehend and visualize. And users may tend to enter wrong values leading to incorrect simulation. Therefore, native support for calculating inertial parameters through SDF specification would enable accurate simulations in simulators that use SDFormat.
+
+## Document summary
+
+The proposal includes the following sections:
+
+* [Motivation](#motivation): A short explanation to provide context reagrding the problem statement and the need for this feature
+* [User Perspective](#user-perspective): Describes the current usage and the proposed usage by describing the terms to be added in the SDFormat specification
+* [Proposed Implmentation](#proposed-implementation): Detailed explanation of the proposed implmentation and modifications to be done in the C++ API of requried libraries like libsdformat. 
+
+## Motivation
+
+Currently, there are 2 major workflows used by the users to obtain the correct inertial parameters of their models:
+
+ * Using CAD softwares like [Fusion360](https://www.autodesk.in/products/fusion-360/overview?term=1-YEAR&tab=subscription) or [Solidworks](https://www.solidworks.com/). Many users design their robot models using such CAD software which usually provide plugins that automatically generates the URDF/SDF for their model. These plugins handle the calculation of the inertial parameters. For eg, Fusion360 provides the [Fusion2URDF](https://github.com/syuntoku14/fusion2urdf) plugin which automatically generates a URDF with all the inertial parameters.
```

Comment:
nit: softwares -> software. Plural of software is software.

Reply:
Corrected in d240e64041c050fe6b5af3c3e1310c907c106625